src/controllers/MigrationController.php

```diff
@@ -409,6 +409,23 @@ public function massDecision($type, $table, $actionMethod)
         }
     }
 
+
+    /**
+     * Lists all Tables in the database.
+     */
+    public function actionList()
+    {
+        $this->stdout("Your Database contains the following tables:\n");
```

Comment:
This is nice idea but maybe this info should go into `else` block so when there are no tables user does not see 

    Your Database contains the following tables:
    Your Database does not contain any tables yet.

but only the second line. What do you think?